src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,103 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static StorageDaoImpl storage;
+    private User userBob;
+    private User user;
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        storage = new StorageDaoImpl();
+        storage.add(new User("BobAlreadyExists", "password", 19));
+    }
```

Comment:
Why do you create storage variable? 
You need to check register method, and now you add a user to storage directly by adding a method to the storage.